content/guides/models/app/settings-page/teams.md

```diff
@@ -112,12 +112,12 @@ The proceeding table lists permissions that apply to all artifacts across a give
 | Permissions      | View-Only | Team Member | Team Admin | 
 | ---------------- | --------- | ----------- | ---------- |
 | View artifacts   | X         | X           | X          |
+| Download artifacts | X        | X           | X          |
 | Create artifacts |           | X           | X          |
 | Delete artifacts |           | X           | X          |
```

Comment:
Aren't these two entriues actually the same permission?

Reply:
These tables are overly verbose so users know exactly what they can and can not do. We list every action, even if it's the same permission.